src/include/kdbprivate.h

```diff
@@ -539,6 +539,54 @@ int elektraKeyLock (Key * key, enum elektraLockOptions what);
 
 ssize_t ksSearchInternal (const KeySet * ks, const Key * toAppend);
 
+/**
```

Comment:
I did not check what the actual size change is, but as long you are the only user such a macro will not hurt.